packaging/suse/amazon-ecs-init.changes

```diff
@@ -23,45 +27,45 @@ Mon Nov 11, 09:00:00 PST 2019 - shugy@amazon.com - 1.33.0-1
 - Cache Agent version 1.33.0
 - Fix destination path in docker socket bind mount to match the one specified using DOCKER_HOST on Amazon Linux 2
 -------------------------------------------------------------------
-Mon Oct 28, 09:00:00 PST 2019 - shugy@amazon.com - 1.32.1-1
+Mon Oct 28, 10:00:00 PDT 2019 - shugy@amazon.com - 1.32.1-1
```

Comment:
just curious - why PST to PDT update?

Reply:
this is done by the changelog.go script the release process says to run.